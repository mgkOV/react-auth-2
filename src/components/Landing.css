.landing-card {
  width: 700px;
  margin: 150px auto;
  color: #fff;
  text-align: center;
  padding: 15px;
}
