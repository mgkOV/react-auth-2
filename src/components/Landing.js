import React, { Component } from 'react';
import './Landing.css';

class Landing extends Component {
  render() {
    return (
      <div className="card medium indigo darken-1 landing-card ">
        <h1>Landing</h1>
      </div>
    );
  }
}

export default Landing;
