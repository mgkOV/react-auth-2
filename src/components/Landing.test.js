import React from 'react';
import { shallow } from 'enzyme';
import Landing from './Landing';

describe('Landing', () => {
  const landing = <Landing />;
  it('should render properly', () => {
    expect(landing).toMatchSnapshot();
  });
});
