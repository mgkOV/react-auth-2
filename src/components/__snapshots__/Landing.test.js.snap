// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Landing should render properly 1`] = `<Landing />`;
